hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -113,3 +113,50 @@ hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
 Note: Only port number from the config is used in this case, 
 whereas the host name in the config `ozone.om.address` is ignored.
 
+## Setting up the ofs
+Please add the following entry to the core-site.xml.
```

Comment:
These sections seem to overlap with #1144. We can simply add a reference here.

Reply:
References to ofs.md have been added.